### ISSUE TITLE
Wordpress no file access

### ISSUE THREAD

**Issue by df-a** [df-a/yggdrasil#2]
Installation won't allow drag and drop plugin installations, nor any kind on non-shell installations of anything files related.

**Comment by df-a** [comment]
#2 is a permissions problem. You have to add: 
- https://thrivethemes.com/tkb_item/sure-want-wordpress-error-message-upload-plugin/  
So to fix permissions, add after wp install, do:  
```terminal
$ WEBROOT='/usr/share/nginx/html'
```
```terminal
$ sudo adduser $USER www-data
```
```terminal
$ sudo chown -R www-data:www-data $WEBROOT
```
```terminal
$ sudo chmod -R g+rwX $WEBROOT
```

**Comment by df-a** [comment]
#2 "please try again" error is also a php memory limit error. So, you have to fix #3 also.

**Comment by df-a** [comment]
Upon install, please check for the following /etc/nginx/global/wordpress.conf

```terminal
location ~* ^.+\.(ogg|ogv|svg|svgz|eot|otf|woff|mp4|ttf|rss|atom|jpg|jpeg|gif|png|ico|zip|tgz|gz|rar|bz2|doc|xls|exe|ppt|tar|mid|midi|wav|bmp|rtf)\$ {
       access_log off; log_not_found off; expires max;
}
```

should look like this

```terminal
location ~* ^.+\.(ogg|ogv|svg|svgz|eot|otf|woff|mp4|ttf|rss|atom|jpg|jpeg|gif|png|ico|zip|tgz|gz|rar|bz2|doc|xls|exe|ppt|tar|mid|midi|wav|bmp|rtf)$ {
  access_log off; log_not_found off; expires max;
}
```
